package.json

```diff
@@ -59,7 +59,8 @@
     "@types/js-cookie": "3.0.6",
     "@types/lodash": "4.17.0",
     "@types/mapbox__mapbox-gl-draw": "^1.4.6",
-    "blitz": "2.0.0-beta.34",
+    "ajv": "8.12.0",
```

Comment:
What does this do? I don't see it being used.

Reply:
was needed by blitz 2.0.8